crates/eframe/src/web/text_agent.rs

```diff
@@ -119,14 +125,20 @@ impl TextAgent {
 
         let Some(ime) = ime else { return Ok(()) };
 
-        let canvas_rect = super::canvas_content_rect(canvas);
-        let cursor_rect = ime.cursor_rect.translate(canvas_rect.min.to_vec2());
+        //let canvas_rect = super::canvas_content_rect(canvas);
+        //let cursor_rect = ime.cursor_rect.translate(canvas_rect.min.to_vec2());
```

Comment:
This is needed for when the canvas is not in the top left. Try `web_demo/multiple_apps.html`, for instance